lib/handlers/actHandler.ts

```diff
@@ -158,7 +158,8 @@ export class StagehandActHandler {
         goal: action,
         steps,
         llmProvider: this.llmProvider,
-        modelName: model,
+        // o1 is overkill for this task + this task uses a lot of tokens. So we switch it 4o
+        modelName: (model === "o1-mini" || model === "o1-preview") ? "gpt-4o" : model,
```

Comment:
not the biggest fan of this approach, but until we find a better one, can we move this into `verifyActCompletion` body instead of having it be a param?